src/Scouting.elm

```diff
@@ -85,29 +85,50 @@ startButton model =
         ]
 
 
+type alias StrModel =
+    { scouter : String
+    , team : String
+    , match : String
+    , driverStation : String
+    , isStarted : Bool
+    }

```

Comment:
Do you really need another `Model`?

Reply:
It's easier than converting to string every time..?

Reply:
> It's easier than converting to string every time..?

Precisely. So you don't need the other one. Just name this `Model` and you're good to go.